README.md

```diff
@@ -1,82 +1,94 @@
+
+Well, because Git is a distributed version control system, each Git repository has a copy of all files in the entire history. As large repositories, aka _monorepos_ grow, Git can struggle to manage all that data. As Git commands like `status` and `fetch` get slower, developers stop waiting and start switching context. And context switches harm developer productivity.
+
+Microsoft Git is focused on addressing these performance woes and making the monorepo developer experience first-class. It does so in part by working with the [GVFS protocol](https://docs.microsoft.com/en-us/azure/devops/learn/git/gvfs-architecture#gvfs-protocol) to prefetch packs of commits and trees and delay downloading of associated blobs. This is required for monorepos using [VFS for Git](https://github.com/microsoft/VFSForGit/blob/master/Readme.md). Additionally, some Git hosting providers support the GVFS protocol instead of the Git-native [partial clone feature](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/).
+
+Downloading and Installing
+=========================================================
+
+If you're working in a monorepo and want to take advantage of Microsoft Git's performance boosts, you can
+download the latest version installer for your OS from the [Releases page](https://github.com/microsoft/git/releases). Alternatively,
+you can opt to install via the command line, using the below instructions for supported OSes:
+
+## Windows
+__Note:__ Winget is still in public preview, meaning you currently [need to take special installation steps](https://docs.microsoft.com/en-us/windows/package-manager/winget/#install-winget).
+
+To install with Winget, run
+
+```shell
+winget install microsoft/git
+```
+
+To upgrade Microsoft Git, use the following Git command, which will download and install the latest release.
+
+```shell
+git update-microsoft-git
+```
+
+You may also be alerted with a notification to upgrade, which presents a single-click process for running `git update-microsoft-git`.
+
+## macOS
+
+To install Microsoft Git on macOS, first [be sure that Homebrew is installed](https://brew.sh/) then install the `microsoft-git` cask with these steps:
+
+```shell
+brew tap microsoft/git
+brew install --cask microsoft-git
+```
+
+To upgrade microsoft/git, you can run the necessary brew commands:
+
+```shell
+brew update
+brew upgrade --cask microsoft-git
+```
+
+Or you can run the `git update-microsoft-git` command, which will run those brew commands for you.
+
+## Linux
+
+For Ubuntu/Debian distributions, `apt-get` support is coming soon. For now, though, please use the most recent [`.deb` package](https://github.com/microsoft/git/releases).
+
+```shell
+wget https://github.com/microsoft/git/releases/download/v2.31.1.vfs.0.1/git-vfs_2.31.1.vfs.0.1.deb microsoft-git.deb
+sudo dpkg -i microsoft-git.deb
```

Comment:
```suggestion
wget -o microsoft-git.deb https://github.com/microsoft/git/releases/download/v2.31.1.vfs.0.1/git-vfs_2.31.1.vfs.0.1.deb
sudo dpkg -i microsoft-git.deb
```

I apologize that I was incorrect in remembering the `wget` `-o` option. I think it is easier to use this so a user only needs to modify the version string in one place.

(This makes me want to create a workflow that triggers on a new release to replace this version string and create a PR. But that can wait until I'm _really_ bored.)